driver-core/src/test/java/com/datastax/driver/core/utils/CassandraVersion.java

```diff
@@ -34,7 +34,12 @@
     /**
      * @return The minor version required to execute this test, i.e. "0"
      */
-    int minor() default 0;
+    int minor() default -1;
+
+    /**
+     * @return Whether or not the exact major version is exactly required to run this test.
+     */
+    boolean[] exactRequired() default {false};
```

Comment:
It's needed to for defaulting the value I believe.   The same thing is done in `@CCMConfig`